src/styles/globals.css

```diff
@@ -0,0 +1,125 @@
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
+
+@import url('https://fonts.googleapis.com/css2?family=Quicksand:wght@300;400;500;600;700&display=swap');
+
```

Comment:
@siroshan Let's add font loading side the app file—if we do this css file has to load first to initiate the font loading—they should load parallelly for a better performance!